compose/compose.go

```diff
@@ -116,6 +136,36 @@ func (c *Compose) MustKill() {
 	}
 }
 
+// GetPublicIPAddressForService returns the IPAddress of a service
+func (c *Compose) GetPublicIPAddressForService(serviceName string) (bool, string) {
```

Comment:
Ahh I see what you mean now by just specifying a "containerport" port. 

I think that makes sense to me - I've always just used the IPAddress (probably too much as I've spent a lot of time with K8s)

I'll have a little think...